data_steward/tools/consolidated_reports/main.py

```diff
@@ -1,11 +1,15 @@
 import json
 import os
 
-import gcs_utils
+from gcloud.gcs import StorageClient
+import app_identity
 from tools.consolidated_reports import query_reports
 from io import open
 
-DRC_BUCKET_PATH = 'gs://%s/' % gcs_utils.get_drc_bucket()
+project_id = app_identity.get_application_id()
+storage_client = StorageClient(project_id)
+drc_bucket = storage_client.get_drc_bucket()
+DRC_BUCKET_PATH = f'gs://{drc_bucket.name}/'
```

Comment:
@lledezma Here is the a suggestion to refactor this 4 lines of repeated code to query_reports.py 
```def get_drc_bucket_path():
        project_id = app_identity.get_application_id()
        storage_client = StorageClient(project_id)
        drc_bucket = storage_client.get_drc_bucket()
        
        return f'gs://{drc_bucket.name}/'```